Added new kb article specifying-binding-cellcontenttemplate-datatable-xaml (#791)
telerik/maui-docs: controls/datagrid/datatable-support.md

> For a sample implementation of the DataGrid commands in a DataTable, see the `DataGrid/DataTable` folder of the [SDKBrowser Demo Application]({%slug sdkbrowser-app%}).

## Binding when Using a CellContentTemplate

> If you have a scenario with a custom template inside the built-in columns&mdash;`CellContentTemplate` and want to bind the property from the business model to an UI elemetn inside the template, you have to use indexer syntax. Review the [Specifying the Binding for a CellContentTemplate in Telerik .NET MAUI DataGrid with DataTable]({%slug specifying-binding-cellcontenttemplate-datatable%}) artivcle for more details how to achieve this sceanrio.

Reviewer: ```suggestion
When you use a `CellContentTemplate` in the built-in columns and you need to bind a property from the business model to a UI element inside the template, you must use indexer syntax. Review the [Specifying the Binding for a `CellContentTemplate` in Telerik .NET MAUI DataGrid with DataTable]({%slug specifying-binding-cellcontenttemplate-datatable%}) article for more details on how to achieve this.
```
